crates/core_arch/src/core_arch_docs.md

```diff
@@ -251,16 +251,6 @@ Next up let's take a look at an example of manually using intrinsics. Here
 we'll be using SSE4.1 features to implement hex encoding.
 
 ```
-# #![cfg_attr(not(dox),feature(stdsimd))]
-# #![cfg_attr(not(dox), no_std)]
-# #[cfg(not(dox))]
-# extern crate std as real_std;
-# #[cfg(not(dox))]
-# extern crate core_arch as std;
-# #[cfg(not(dox))]
-# #[macro_use(is_x86_feature_detected)]
-# extern crate std_detect;
```

Comment:
Should we be doing this everywhere ? There are many doc examples where we do this dance.

Reply:
Yeah I don't really know what was happening to have the doctests fail, but this is all in std now so none of this was necessary any more